RadixWallet/Features/DerivePublicKeys/DerivePublicKeys.swift

```diff
@@ -210,6 +212,7 @@ public struct DerivePublicKeys: Sendable, FeatureReducer {
 					)
 				case let .ledger(ledgerFactorSource):
 					return deriveWith(ledgerFactorSource: ledgerFactorSource, state)
+				default: fatalError("DISCREPANCY: Found non .device | .ledger factor source. A real world user cannot possible have this.")
```

Comment:
This is safe for our users, it is not possible for our users to create any other factor source than `DeviceFactorSource` or `LedgerFactorSource`.